### DISCUSSION TITLE
Disable export declaration

### DISCUSSION THREAD

**Discussion by vimtor** [eslint/eslint#15106] [Q&A]
Hi there! Is there any way to disable or autofix this:

```js
export const foo = 'bar';
```

To this:

```js
const foo = 'bar';
export { foo };
```

I've been searching plugins and rules and didn't find anything.

**Comment by aladdin-add** [comment]
it can be achieved by https://eslint.org/docs/rules/no-restricted-syntax: [an example](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLXJlc3RyaWN0ZWQtc3ludGF4OiBbXCJlcnJvclwiLCBcIkV4cG9ydE5hbWVkRGVjbGFyYXRpb24+VmFyaWFibGVEZWNsYXJhdGlvblwiXSAqL1xuZXhwb3J0IGNvbnN0IGZvbyA9IDA7XG5cbnZhciBiYXIgPSAwO1xuZXhwb3J0IHtiYXJ9Iiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoxMiwic291cmNlVHlwZSI6Im1vZHVsZSIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6e30sImVudiI6e319fQ==)
